lib/utils/parseRequest.js

```diff
@@ -1,3 +1,6 @@
 module.exports = rawRequest => {
-
+  const parsedRequest = rawRequest.split('\r\n');
+  const [method, path] = (parsedRequest[0].split(' '));
+  const body = parsedRequest[2];
+  return { method,  path, body };
```

Comment:
Make sure to clean up extra whitespace:

```suggestion
  return { method, path, body };
```

Reply:
I will pay more attentions to it.